plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -43,7 +43,7 @@ class RayAutoScalingMode(Enum):
 
 
 @dataclass
-class RayDockerConf(Dict[str, Any]):
+class RayDockerConf:
```

Comment:
Good, but there are many others:

```
class RayInitConf(Dict[str, Any]):
```

Can you try to fix all of them?
if it means adding more variables to the config it's fine.

Reply:
Ok i think now they are all fixed.